django/db/models/query.py

```diff
@@ -322,6 +322,9 @@ def aggregate(self, *args, **kwargs):
         """
         if self.query.distinct_fields:
             raise NotImplementedError("aggregate() + distinct(fields) not implemented.")
+        if (not all(isinstance(kwarg, (BaseExpression, Combinable)) for kwarg in kwargs.values()) or
+                not all(isinstance(arg, (BaseExpression, Combinable)) for arg in args)):
+            raise TypeError('Arguments must all be expressions.')
```

Comment:
I'd say something like "Queryset.aggregate() arguments must be expressions." Not worth it to list the invalid args?